packages/blade/src/components/Icons/Edit2Icon/Edit2Icon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Svg, Path } from '../_Svg';
+import type { IconComponent } from '..';
+import useIconProps from '../useIconProps';
+
+const Edit2Icon: IconComponent = ({ size, color }) => {
```

Comment:
Do we need any changes in the existing EditIcon? @divyanshu013 @kamleshchandnani 

Else i can simply delete Edit2Icon and Edit3Icon. 

Ref thread: https://razorpay.slack.com/archives/CMQ3RBHEU/p1672817807133059?thread_ts=1672817419.176859&cid=CMQ3RBHEU

<img width="355" alt="image" src="https://user-images.githubusercontent.com/35374649/210962945-cba08351-c436-44be-989a-9daa1d9e54a2.png">

Reply:
I think consensus is to avoid having duplicate names so we can have a single variant for edit